helmfile/values/fluentd-aggregator.yaml.gotmpl

```diff
@@ -1,98 +0,0 @@
-image:
-  repository: quay.io/fluentd_elasticsearch/fluentd
-  tag: v2.9.0
-
-resources: {{- toYaml .Values.fluentd.aggregator.resources | nindent 2  }}
-nodeSelector: {{- toYaml .Values.fluentd.aggregator.nodeSelector | nindent 2  }}
-affinity: {{- toYaml .Values.fluentd.aggregator.affinity | nindent 2  }}
-tolerations: {{- toYaml .Values.fluentd.aggregator.tolerations | nindent 2 }}
-
-plugins:
-  enabled: true
-  pluginsList:
-    {{ if eq .Values.objectStorage.type "s3" -}}
-    # TODO: pin version
-    - fluent-plugin-s3
-    {{ else if eq .Values.objectStorage.type "gcs" -}}
-    - digest-crc -v "0.5.1"
-    - fluent-plugin-gcs
-    {{- end }}
```

Comment:
Make sure we don't loose this S3 or GCS logic in the new setup.

Reply:
> Make sure we don't loose this S3 or GCS logic in the new setup.

Done!